logging/bin/deploy_logging_opensearch.sh

```diff
@@ -0,0 +1,115 @@
+#! /bin/bash
+
+# Copyright © 2020, SAS Institute Inc., Cary, NC, USA.  All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+# identify backend search provider
+export LOG_SEARCH_BACKEND="OPENSEARCH"
+
+cd "$(dirname $BASH_SOURCE)/../.."
+source logging/bin/common.sh
+
+# Confirm NOT on OpenShift
+if [ "$OPENSHIFT_CLUSTER" == "true" ]; then
+  if [ "${CHECK_OPENSHIFT_CLUSTER:-true}" == "true" ]; then
+    log_error "This script should NOT be run on OpenShift clusters"
+    log_error "Run logging/bin/deploy_logging_open_openshift.sh instead"
```

Comment:
Good catch.  I was/am holding off on making all of the OpenShift related changes until I can complete my research.  However, there's no reason to hold off on fixing this reference here since it would likely be missed later.  Will fix and push.